jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/ColorSpinnersTests.java

```diff
@@ -0,0 +1,64 @@
+package io.github.epam.bootstrap.tests.common;
+
+import com.epam.jdi.light.ui.bootstrap.elements.common.ColorSpinner;
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.*;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.testng.Assert.assertEquals;
+
+public class ColorSpinnersTests extends TestsInit {
+
+    @DataProvider(name = "Color Spinners")
+    public static Object[][] colorSpinners() {
+        return new Object[][]{
+                {spinnerWithTextPrimary, "text-primary"},
+                {spinnerWithTextSecondary, "text-secondary"},
+                {spinnerWithTextDanger, "text-danger"},
+                {spinnerWithTextWarning, "text-warning"},
+                {spinnerWithTextSuccess, "text-success"},
+                {spinnerWithTextInfo, "text-info"},
+                {spinnerWithTextLight, "text-light"},
+                {spinnerWithTextDark, "text-dark"}
+        };
+    }
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test(dataProvider = "Color Spinners")
+    public void getSpinnerColorTests(ColorSpinner colorSpinner, String color) {
+        assertEquals(colorSpinner.getColor(), color);
```

Comment:
it's better to rewrite to: colorSpinner.is().color(color); (for example by creating ColorSpinnerClass). it allows us to use jdi style in tests instead of testng